src/applications/vaos/services/appointment/transformers.js

```diff
@@ -425,7 +422,11 @@ function setContained(appt) {
           resourceType: 'HealthcareService',
           characteristic: [
             {
-              coding: getVideoType(appt),
+              coding: [
+                {
+                  system: 'VVS',
```

Comment:
Technically there should be a code here, but for requests we don't have or need a kind.